### PR TITLE
[jalalRifaat][Vendeur Crud][Tp1-22]

### PR DESCRIPTION
**Pull request by RifaatJalal**
Pull request

### CODE REVIEW

File: src/main/java/com/groupinfo5ltd/tastyManagement/entity/Vendeur.java [hunk 1]
```diff
@@ -33,5 +33,37 @@
 	private String lastName; 
 	
 	@OneToMany(mappedBy = "vendeur")
-	private Set<Vente> ventesVendeur = new HashSet<>(); 
+	private Set<Vente> ventesVendeur = new HashSet<>();
+
+	public long getId() {
```
Comment (medkhabt) [inline src/main/java/com/groupinfo5ltd/tastyManagement/entity/Vendeur.java]: had les getters w setters ki generaw automatiquement b @Data foug mn la class, man7tajouch nzidouhoum

File: src/main/java/com/groupinfo5ltd/tastyManagement/service/impl/VendeurService.java [hunk 2]
```diff
@@ -0,0 +1,69 @@
...
+	IVendeurRepository vendeurRepository;
+
+	@Override
+	public Vendeur ajouterVendeur(Vendeur vendeur) {
+		Vendeur vendeurToCreate = null;
+		try {
+			vendeurToCreate = vendeurRepository.save(vendeur);
+			log.info("CREATE A new Vendeur" + vendeur.toString());
+		} catch (Exception e) {
+			log.error("CREATE A new Vendeur" + vendeur.toString());
```
Comment (medkhabt) [inline src/main/java/com/groupinfo5ltd/tastyManagement/service/impl/VendeurService.java]: f l message dial had log 7sn ktbou
`log.error("UNABLE to CREATE A new vendeur Caused by: " + e.toString())`

File: src/main/java/com/groupinfo5ltd/tastyManagement/entity/Vendeur.java (hunk 1 above)
Comment (RifaatJalal) [reply src/main/java/com/groupinfo5ltd/tastyManagement/entity/Vendeur.java]: d'accord hit ma3ndich ana lombok mhm andir bhal code dyalk wakha kaytl3li ana error f getId() ila madrtch getters

File: src/main/java/com/groupinfo5ltd/tastyManagement/service/impl/VendeurService.java (hunk 2 above)
Comment (RifaatJalal) [reply src/main/java/com/groupinfo5ltd/tastyManagement/service/impl/VendeurService.java]: rah dert error donc ghatban error :""CREATE A new Vendeur" + vendeur.toString()"

Comment (medkhabt) [reply src/main/java/com/groupinfo5ltd/tastyManagement/service/impl/VendeurService.java]: ahh, 3ndk s7

**Pull request merged**